### PR TITLE
Docs: add server view folder

### PR DESCRIPTION
**Pull request by oddlittlebird**
As part of my continuing quest to bring some coherence to the Administration folder, I moved three items that have a similar theme (see what's going on with your Grafana instance) into one folder. I also updated the weights for some of the other items, as most of them were set to `8`.

The first two topics in the list, Change your password and Change preferences, will be removed in later PRs. Their content is being rewritten and expanded.

### CODE REVIEW

File: docs/sources/administration/view-server/view-server-settings.md [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
 keywords = ["grafana", "configuration", "server", "settings"]
-weight = 350
+aliases = ["/docs/grafana/latest/admin/view-server-settings/"]
+weight = 300
 +++
 
 # View Grafana server settings
 
-If you are a Grafana server admin, then you can use the Settings tab to view the settings applied to your Grafana server by the [Configuration]({{< relref "configuration.md#config-file-locations" >}}) file and any environmental variables.
+If you are a Grafana server admin, then you can use the Settings tab to view the settings applied to your Grafana server by the [Configuration]({{< relref "../configuration.md#config-file-locations" >}}) file and any environmental variables.
```
Comment (osg-grafana) [inline docs/sources/administration/view-server/view-server-settings.md:11]:
```suggestion
If you are a Grafana server administrator, use the Settings tab to view the settings that are applied to your Grafana server via the [Configuration]({{< relref "../configuration.md#config-file-locations" >}}) file and any environmental variables.
```

File: docs/sources/administration/view-server/view-server-settings.md [hunk 2]
```diff
@@ -2,14 +2,15 @@
...
 
-> **Note:** Only Grafana server admins can access the **Server Admin** menu. For more information about about admin permissions, refer to [Grafana server admin]({{< relref "../permissions/_index.md" >}}).
+> **Note:** Only Grafana server admins can access the **Server Admin** menu. For more information about about admin permissions, refer to [Grafana server admin]({{< relref "../../permissions/_index.md" >}}).
```
Comment (osg-grafana) [inline docs/sources/administration/view-server/view-server-settings.md:13]:
```suggestion
> **Note:** Only Grafana server administrators can access the **Server Admin** menu. For more information about about administrative permissions, refer to [Grafana server admin]({{< relref "../../permissions/_index.md" >}}).
```

File: docs/sources/administration/view-server/view-server-stats.md [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+aliases = ["/docs/grafana/latest/admin/view-server-stats/"]
+weight = 400
 +++
 
 # View Grafana server stats
 
 If you are a Grafana server admin, then you can view useful statistics about your Grafana server in the Stats tab.
 
-> **Note:** Only Grafana server admins can access the **Server Admin** menu. For more information about about admin permissions, refer to [Grafana server admin]({{< relref "../permissions/_index.md" >}}).
+> **Note:** Only Grafana server admins can access the **Server Admin** menu. For more information about about admin permissions, refer to [Grafana server admin]({{< relref "../../permissions/_index.md" >}}).
```
Comment (osg-grafana) [inline docs/sources/administration/view-server/view-server-stats.md:12]:
```suggestion
> **Note:** Only Grafana server administrators can access the **Server Admin** menu. For more information about about administrative permissions, refer to [Grafana server admin]({{< relref "../../permissions/_index.md" >}}).
```

**Review by osg-grafana — CHANGES_REQUESTED** [review]

**Review by osg-grafana — APPROVED** [review]

File: docs/sources/administration/view-server/_index.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
++++
+title = "View server"
+weight = 100
++++
+
+# View server information
+
+This setting contains information about tools that Grafana Server Admins can use to learn more about their Grafana servers.
```
Comment (achatterjee-grafana) [inline docs/sources/administration/view-server/_index.md:8]: Is this a setting or a capability?

Comment (oddlittlebird) [reply docs/sources/administration/view-server/_index.md:8]: This is collecting three extant topics that all fall under the general category of "see what is going on with your Grafana instance."

**Review by achatterjee-grafana — APPROVED** [review]
Looks good. We can review again once all the PRs are merged, post 7.4 release.

**Pull request merged**